packages/proto-signing/src/codec/cosmos/bank/v1beta1/bank.ts

```diff
@@ -0,0 +1,650 @@
+/* eslint-disable */
```

Comment:
(comment not related to the actual line, but to Will's original comment, I think having threads make things clearer).

> It doesn't seem possible to generate types except via .proto files, so we'd be giving up the decorator approach. We also can't use the more manual approach from demo.ts.

This was what I was initially proposing, and what I'm exploring inside regen-js. Happy to discuss more about it during our next call.